### PR TITLE
change "and" to "or" to get correct aggregates and unique filtering 

### LINKED ISSUE #1678: Receipts data: Unique filter behaving weird
@fec-issue-bot commented on [Fri May 20 2016](https://github.com/18F/FEC/issues/325)

## What were you trying to do and how can we improve it?
Here is a good example of the UNIQUE issue I mentioned before. Steve Aarnio has made four direct contributions to ACTBLUE and some earmarked contributions to Bernie through ACTBLUE. When you hit UNIQUE all the earmarked contributions go away.  

## Tell us about yourself
Jeff Chumley
Time traveler

## Details
* URL: https://beta.fec.gov/data/receipts/?committee_id=C00401224&amp;contributor_name=steve+aarnio&amp;is_individual=true
* User Agent: Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36

### PR DESCRIPTION
**Pull request by LindsayYoung**
the check for earmark is for exclusion with the "and", we were filtering out all the small donors.

closes #1678

There is also some data that needs to be found, but that is a separate issue.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
Thanks again to @jwchumley for taking the time to walk through this

**Comment by jmcarp** [comment]
Does this line need to change too?
https://github.com/18F/openFEC/blob/develop/webservices/docs.py#L305

**Comment by jwchumley** [comment]
Probably. I'm not sure what the reason is for excluding earmarks that are 
under $200.

**Comment by LindsayYoung** [comment]
Good point, I'll update that.

I think the title foir that section should say:
Methodology for determining unique, individual contributions

And that line should say:
Of those transactions,[under $200, and having "earmark" in the memo text OR  transactions having the codes 11A, 12, 17, 17A, or 18], we then want to exclude earmarks.

**Comment by LindsayYoung** [comment]
Tests pass for me locally, I think it is because I am using the new test subset in https://github.com/18F/openFEC/pull/1628

**Comment by LindsayYoung** [comment]
recreated the issue by merging in develop

**Comment by LindsayYoung** [comment]
I tried merging in develop and running on new test data. Still fails as a test but runs if you try it on my test data set. I am going to try the Travis steps exactly and see if I can reproduce the error.

**Comment by noahmanger** [comment]
Are we trying to get this in to this release? Seems like we should.

**Comment by LindsayYoung** [comment]
@noahmanger Yes, we are. The test is failing and we are having issues debugging it.

**Comment by jontours** [comment]
@ccostino Ready to merge once you've reviewed.

File: tests/test_integration.py [hunk 1]
```diff
@@ -409,4 +409,4 @@ def test_update_aggregate_purpose_existing(self):
         manage.update_aggregates()
         db.session.refresh(existing)
         self.assertEqual(existing.total, total + 538)
-        self.assertEqual(existing.count, count + 1)
+        self.assertEqual(existing.count, count + 1)
```
Comment (ccostino) [inline tests/test_integration.py]: Being a bit of a nit-pick here, but could we get a newline at the end of this file just to cut down on linting warnings? Thanks!

**Pull request merged**